draft-jholland-mops-taxonomy.md

```diff
@@ -174,53 +184,53 @@ here, but it seems worth making the point that demand projections can
 or by observing the successful application-layer download speed,
-then chooses a video bitrate that fits within that bandwidth,
+then chooses a video bitrate (among the limited number of available options) that fits within that bandwidth,
 typically adjusting as changes in available bandwidth occur in
-the network.
+the network or changes in capabilities occur in the player (such as available memory, CPU, display size, etc.).
 
-The choice of bit-rate occurs within the context of optimizing for
+The choice of bitrate occurs within the context of optimizing for
 some metric monitored by the video player, such as highest achievable
 video quality, or lowest rate of expected rebuffering events.
 
 ##Segmented Delivery
 
-ABR strategies are commonly implemented by video players using HLS
-{{RFC8216}} or DASH {{DASH}} to perform a reliable segment delivery
-of video data over HTTP.  Different player implementations and
+ABR playback is commonly implemented by video players using HLS
+{{RFC8216}} or DASH {{DASH}} to perform a reliable segmented delivery
+of video data over HTTP. Different player implementations and
 receiving devices use different strategies, often proprietary
-algorithms, to perform the bit-rate selection and available
-bandwidth estimation.
+algorithms (called rate adaptation or bitrate selection algorithms), to perform available
+bandwidth estimation/prediction and the bitrate selection. Most players only use passive observations, i.e., they do not generate probe traffic to measure the available bandwidth. 
 
-This kind of bandwidth-detection system can experience trouble in
+This kind of bandwidth-measurement systems can experience trouble in
 several ways that can be affected by networking design choices.
 
-###Idle Time Between Segments
+###Idle Time between Segments
 
-When the bit-rate selection is successfully chosen below the
+When the bitrate selection is successfully chosen below the
 available capacity of the network path, the response to a
-segment request will complete in less absolute time than the
-video bit-rate speed.
-
-The resulting idle time within the connection carrying the
+segment request will typically complete in less absolute time than the
+duration of the requested segment. The resulting idle time within the connection carrying the
 segments has a few surprising consequences:
 
  * Mobile flow-bandwidth spectrum and timing mapping.
 
- * TCP Slow-start when restarting after idle requires multiple
+ * TCP slow-start when restarting after idle requires multiple
    RTTs to re-establish a throughput at the network's available
-   capacity.  On high-RTT paths or with small enough segments,
+   capacity. On high-RTT paths or with small enough segments,
    this can produce a falsely low application-visible measurement
    of the available network capacity.
 
-###Head of Line Blocking
+A detailed investigation of this phenomenon is available in {{NOSSDAV12}}.
```

Comment:
Any chance we could include some of the key observations for this?  There's a paywall on the reference...

Reply:
Sure, I will add this in the next revision.